cottontaildb_client/cottontaildb_cli.py

```diff
@@ -123,12 +128,59 @@ def schema(client, args):
 
 def entity(client, args):
     command = args.subcommand
+    response = None
     if command == 'drop':
         response = client.drop_entity(args.schema_name, args.entity_name)
-        print(response)
     elif command == 'about':
         response = client.get_entity_details(args.schema_name, args.entity_name)
-        print(response)
+    elif command == 'preview':
+        response = client.sample_entity(args.schema_name, args.entity_name, limit=args.limit)
+    if response is not None:
+        print(format_response(response))
+
+
+def format_response(response):
+    """Formats the response object as a string.
+
+    Args:
+        response (object): The object to format.
+    Returns:
+        str: The formatted response as a string.
+    """
+    if type(response) is list and all([type(item) is dict for item in response]) and len(response) > 0:
+        return format_as_table(response)
+    else:
+        return response.__repr__()
+
+
+def format_as_table(dict_list):
+    """Formats a list of dictionaries as a table.
+
+    Args:
```

Comment:
This project uses the Epytext docstring format (have a look at cottontaildb_client documentation or [this article](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats) as an example). While we may migrate to the reStructuredText format at some point to adhere to PEP 287, please convert these docstrings to Epytext for the time being.